### PR TITLE
small modifications to the sample code

### PR DESCRIPTION
**Pull request by leighwaller**

### CODE REVIEW

File: MessageMedia.Api/MessageMediaSoapService.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ public partial class MessageMediaSoapService : System.Web.Services.Protocols.Soa
     
     /// <remarks/>
     public MessageMediaSoapService() {
-        this.Url = "https://soap.m4u.com.au";
+        this.Url = "https://soap.dev.m4u.com.au";
```
Comment (ghost) [inline MessageMedia.Api/MessageMediaSoapService.cs]: .dev?

File: MessageMedia.Api.Console/Program.cs [hunk 2]
```diff
@@ -114,6 +113,8 @@ public static void SendScheduledMessage()
 
                 var result = client.SendScheduledMessage("", recipient1, message, messageId, messageScheduledDateTime);
                 DisplaySendMessageResult(result);
+                
+                DeleteScheduledMessages(new List<uint>{ messageId });
```
Comment (Ox5f3759df) [inline MessageMedia.Api.Console/Program.cs]: I don't think this is a good idea given that it's within an example where you Send the scheduled message.
Probably better to include a separate example where it schedules then deletes.

File: README.md [hunk 3]
```diff
@@ -35,9 +35,13 @@ Other available method calls:
 * SendMessage();
 * SendScheduledMessage();
 * ConstructAndSendBatchMessage();
-* CheckReplies(); 
+* CheckReplies();
+* ConfirmReplies();
 * CheckReports();
-
+* ConfirmReports();
+* DeleteScheduledMessages();
```
Comment (Ox5f3759df) [inline README.md:42]: I don't think this will work.

File: MessageMedia.Api/MessageMediaSoapService.cs (hunk 1 above)
Comment (leighwaller) [inline MessageMedia.Api/MessageMediaSoapService.cs]: ahh shit i forgot to undo it after my testing

File: README.md (hunk 3 above)
Comment (leighwaller) [inline README.md:42]: why is that?

File: MessageMedia.Api.Console/Program.cs (hunk 2 above)
Comment (leighwaller) [inline MessageMedia.Api.Console/Program.cs]: yeh possibly, i was being a bit lazy. Although i did update the documentation to let them know what it was doing

File: README.md (hunk 3 above)
Comment (elruwen) [inline README.md:42]: Looks good to me.

**Pull request merged**